CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 > No unreleased changes yet
 
+### Added
+
+- Added a new feature to the library: `fetchEvents` can now be used to fetch events for a specified zkApp from a GraphQL endpoint that implements the schema specified [here](https://github.com/o1-labs/Archive-Node-API/blob/efebc9fd3cfc028f536ae2125e0d2676e2b86cd2/src/schema.ts#L1).
```

Comment:
I would also briefly mention the new addition to the `Mina.Network` API and the differences between the mina node endpoint and the archive endpoint